bin/desi_purge_night

```diff
@@ -78,3 +78,5 @@ if __name__ == '__main__':
 
     log.info(f"Done purging {specprod} night {night}")
 
+    if not args.not_dry_run:
```

Comment:
For readability could I suggest using `dry_run` (which is already defined as `not args.not_dry_run`?